tox.ini

```diff
@@ -30,9 +30,8 @@ setenv =
 install_command= python -m pip install {opts} {packages}
 commands =
     pip --version
-    coverage run -m pytest --strict --doctest-modules {posargs:tests/ piptools/}
-    coverage report -m
-    coverage html
+    pytest --strict --doctest-modules --cov piptools --cov tests \
+        --cov-report html --cov-report term-missing {posargs:tests/ piptools/}
```

Comment:
Just `--cov` should be enough - then it uses the config (which as source="." though).  I think those should be in line.

Reply:
Good call!